Improve docs about integrations (#882)
Uninett/Argus: docs/integrations/glue-services/index.rst

.. index::
   integration; glue service
   glue service

=========================================================
Glue services: Integrating monitoring software with Argus
=========================================================

A *glue service* is the name we've given to any piece of software capable of
receiving alerts from your monitoring system and translating these into
incident state changes that are communicated to the Argus API server.

In essence, to integrate your existing monitoring systems with Argus means to
ensure your Argus incident database is synchronized with the alerts stemming
from your monitoring systems.

A glue service for your particular monitoring system may already exist, or you
may need to write your own.

Existing glue services
======================

1st party glue services
-----------------------

Network Administration Visualized
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Source: https://github.com/Uninett/nav-argus-glue.

Reviewer: This also reminds me that the README of this plugin should be updated to include install instructions for installing directly from PyPI.  "Regular" users shouldn't need to care about using git to fetch the source code and install something :)

A *glue service* is the name we've given to any piece of software capable of
receiving alerts from your monitoring system and translating these into
incident state changes that are communicated to the Argus API server.

In essence, to integrate your existing monitoring systems with Argus means to
ensure your Argus incident database is synchronized with the alerts stemming
from your monitoring systems.

A glue service for your particular monitoring system may already exist, or you
may need to write your own.

Existing glue services
======================

Glue services maintained by Argus developers
--------------------------------------------

Network Administration Visualized
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

| Source: https://github.com/Uninett/nav-argus-glue.
| PyPI: `nav-argus-glue <https://pypi.org/project/nav-argus-glue/>`_

This is an extension of NAV, runs on the same server, and uses the NAV config
files.

Mist Systems Administration (Juniper)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

| Source: https://gitlab.sikt.no/cnaas/mist-argus

This will be available on PyPI after more polish is done.

Other glue services
-------------------

NAGIOS
~~~~~~

>   * There is a python script at https://github.com/SUNET/nagios-argus-glue. It

Reviewer 2: ```suggestion
  * There is a Python script at https://github.com/SUNET/nagios-argus-glue. It
```

.. index::
   integration; glue service
   glue service

=========================================================
Glue services: Integrating monitoring software with Argus
=========================================================

A *glue service* is the name we've given to any piece of software capable of
receiving alerts from your monitoring system and translating these into
incident state changes that are communicated to the Argus API server.

In essence, to integrate your existing monitoring systems with Argus means to
ensure your Argus incident database is synchronized with the alerts stemming
from your monitoring systems.

A glue service for your particular monitoring system may already exist, or you
may need to write your own.

Existing glue services
======================

> 1st party glue services

Reviewer 2: ```suggestion
Glue services maintained by Argus developers
```

-----------------------

Network Administration Visualized
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

| Source: https://github.com/Uninett/nav-argus-glue.

> | Pypi: `nav-argus-glue <https://pypi.org/project/nav-argus-glue/>`_

Reviewer 2: ```suggestion
| PyPI: `nav-argus-glue <https://pypi.org/project/nav-argus-glue/>`_
```

This is an extension of NAV, runs on the same server, and uses the NAV config
files.

Mist Systems Administration (Juniper)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

| Source: https://gitlab.sikt.no/cnaas/mist-argus

> This will be available on Pypi after more polish is done.

Reviewer 2: ```suggestion
This will be available on PyPI after more polish is done.
```

Other glue services
-------------------

NAGIOS
~~~~~~

>   * There's a python script at https://github.com/SUNET/nagios-argus-glue. It

Reviewer 2: ```suggestion
  * There is a Python script at https://github.com/SUNET/nagios-argus-glue. It
```

    didn't scale to SUNET's needs.

>   * There's the code for a tiny rust binary at https://github.com/SUNET/nglue

Reviewer 2: ```suggestion
  * There is the code for a tiny rust binary at https://github.com/SUNET/nglue
```

    that is run by NAGIOS. It sends alerts to
    https://github.com/SUNET/nglue-api, designed to run in a container. The

>     nglue-api pre-filters what to send to argus.

Reviewer 2: ```suggestion
nglue-api pre-filters what to send to Argus.
```

.. note::

>    Open an issue (`Github <https://github.com/Uninett/Argus/issues>`_) to have

Reviewer 2: ```suggestion
   Open an issue on `Github <https://github.com/Uninett/Argus/issues>`_ to have
```

may need to write your own.

Existing glue services
======================

> 1st party glue services

Author: Yes. Suggestions to better wording welcome. When we have enough we should probably use a table, we could then have this as a column with a check-mark in it or something.

> 1st party glue services

Author: Johanna had a suggestion which I used everywhere
